<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QWidget" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>464</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabConnection">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="conn">
      <attribute name="title">
       <string>Connection 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ConnectionForm" name="connectionForm_1" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conn2">
      <attribute name="title">
       <string>Connection 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkConn_2">
         <property name="text">
          <string>Enable connection 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ConnectionForm" name="connectionForm_2" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conn3">
      <attribute name="title">
       <string>Connection 3</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkConn_3">
         <property name="text">
          <string>Enable connection 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ConnectionForm" name="connectionForm_3" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConnectionForm</class>
   <extends>QWidget</extends>
   <header>customwidgets/connectionform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
